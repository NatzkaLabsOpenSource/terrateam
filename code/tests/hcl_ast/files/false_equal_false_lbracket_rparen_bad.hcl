false = false [)
---
bar = false [)
---
bar = false [}
---
bar = bar [)
---
bar = 1 [)
---
bar = "foo" [)

