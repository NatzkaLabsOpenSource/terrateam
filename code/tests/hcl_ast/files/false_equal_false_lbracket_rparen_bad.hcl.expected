Expected expression in index operator, found symbol.
